src/data_provider/src/processes/processSolarisWrapper.hpp

```diff
@@ -0,0 +1,238 @@
+/*
+ * Wazuh SYSINFO
+ * Copyright (C) 2015, Wazuh Inc.
+ * March 9, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#ifndef _PROCESS_SOLARIS_WRAPPER_H
+#define _PROCESS_SOLARIS_WRAPPER_H
+
+#include <fstream>
+#include <map>
+
+#ifdef WAZUH_UNIT_TESTING
+#include "../tests/sysInfoProcessesSolaris/sysInfoSolarisProcesses_test.h"
```

Comment:
I think the include is to use the mocked structs, but I'm not sure. Wouldn't it be possible to use gmock, since there's a wrapper interface that we can instantiate from?

Reply:
This pseudo-constant `KBYTES_PER_PAGE` is calculated and in the testing universe using static data it is enough to define the correct value